index.html

```diff
@@ -240,6 +250,60 @@ <h2 class="my-project-name">Multi-Post Stories</h2>
       </div>
     </div>
   </section>
+
+  <div class="contact-form">
+
+      <h3 class="contact-form-title">CONTACT US</h3>
+
+      <p class="contact-form-title">If you have an application you are interested in developing, a feature that you
+         need built or a project that needs coding. I’d love to help with it</p>
+
+    <form action="https://formspree.io/f/moqrjakr" method="post">
+      
+         <input type="text" name="user-first-name" placeholder="Text Input" maxlength="30" required>
+      
+         <input type="email" name="user-email" placeholder="jeremy@gmail.com" class="name-ph-change" 
+         pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
+       
+         <textarea name="user-message" placeholder="Enter text here" class="name-ph-text-area-change" maxlength="500" required></textarea>
+
+        <button type="submit">Send</button>
+
+
+     </form>
+
+  </div>
+
+  <img
+    src="./images/Rectangle_42.svg"
+    class="footer-rectangle-separator"
+    alt="Section separator image"
+  />
```

Comment:
- [ ] The footer of your project is shown like this 
![image](https://user-images.githubusercontent.com/28109626/149242888-57e04613-3fe6-4239-945f-f69c0114e754.png)
Please let's fix this, in order to follow the Figma template as close as possible.

Reply:
As I was trying to use an image as the section separator and it make it difficult to accommodate in the layout I changed the separator to an HTML <hr> tag and it work very well.